MonoGame.Framework/Utilities/OS.cs

```diff
@@ -0,0 +1,46 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace MonoGame.Utilities
+{
+    /// <summary>
+    /// Defines the operating system.
+    /// </summary>
+    public enum OS
```

Comment:
> If you want to check at runtime for Windows/Mac/Linux, it's as simple as:

That's not "simple" by any means, or reliable enough.  Because Mono gets it wrong, we're supposed to magically know the multi-line work-around that depends on an arbitrary folder existing?

Using `#if` goes against the entire point of DesktopGL using the same executable across Windows, Mac and Linux.  And I believe there will always be a need to know which platform you are running on because there will always be something different on each platform, even just some integration with the OS or similar.